charts/newrelic-logging/templates/_helpers.tpl

```diff
@@ -153,12 +153,15 @@ Returns fargate
 Returns lowDataMode
 */}}
 {{- define "newrelic-logging.lowDataMode" -}}
-{{- if .Values.global }}
-  {{- if .Values.global.lowDataMode }}
-    {{- .Values.global.lowDataMode -}}
-  {{- end -}}
-{{- else if .Values.lowDataMode }}
+{{/* `get` will return "" (empty string) if value is not found, and the value otherwise, so we can type-assert with kindIs */}}
+{{- if (get .Values "lowDataMode" | kindIs "bool") -}}
   {{- .Values.lowDataMode -}}
```

Comment:
If `lowDataMode: false`, this will return `"false"` as a string, which is (if I'm not mistaken) evaluated as true when you do

```
{{- if include "newrelic-logging.lowDataMode" . }}
```

Reply:
You are right. Solved.